backend-project/small_eod/cases/serializers.py

```diff
@@ -51,7 +49,7 @@ def create(self, validated_data):
         responsible_user = validated_data.pop("responsible_user")
         notified_user = validated_data.pop("notified_user")
         feature = validated_data.pop("feature")
-        case = Case.objects.create(**validated_data)
+        case = super().create(validated_data)
```

Comment:
Zapewne mamy identyczny problem we wszystkich serializatorach.

Reply:
Zmienione